daemon/logic/worklog.go

```diff
@@ -242,17 +270,257 @@ func (h *inviteApproveHandler) resolve(ctx context.Context, n *observer.Notifier
 	if err != nil {
 		return nil, err
 	}
+
+	return &apitypes.WorklogResult{
+		ID:      item.ID,
+		State:   apitypes.SuccessWorklogResult,
+		Message: "User invite approved and finalized.",
+	}, nil
+}
+
+type keyringMembersHandler struct {
+	engine *Engine
+}
+
+func (keyringMembersHandler) resolveErr() string {
+	return "Error adding user(s) to keyring"
+}
+
+func (h *keyringMembersHandler) list(ctx context.Context, org *envelope.Unsigned) ([]apitypes.WorklogItem, error) {
+	projects, err := h.engine.client.Projects.List(ctx, org.ID)
 	if err != nil {
-		return &apitypes.WorklogResult{
-			ID:      item.ID,
-			State:   apitypes.ErrorWorklogResult,
-			Message: "Error approving invite: " + err.Error(),
-		}, nil
+		return nil, err
+	}
+
+	// Find all of the credential graphs in this org.
```

Comment:
Thoughts? Perhaps you wanted to standardize on the the credentialgraph endpoint for caching?